hw/top_earlgrey/data/autogen/top_earlgrey.gen.hjson

```diff
@@ -7268,7 +7268,7 @@
       }
       base_addrs:
       {
-        rom: 0x00008000
+        rom: 0x00010000
```

Comment:
is it possible/would it be better to start the ROM at 0x0, then the rv_dm mem would not move?

Reply:
I think we prefer not to have anything map to 0x0, so it can be an obvious failure for NULL references.

Reply:
Where is the RvDM RAM located?  I seem to recall that it is at 0x10000.

Reply:
With this PR, it was moved up to `0x00080000`. I'm surprised there aren't any obvious signs of changes to the DM firmware or some Ibex vector or something, though... (nothing I spotted, at least)

Reply:
Ah, it plugs in via a parameter, which references the xbar / tlul generation script outputs:

https://github.com/lowRISC/opentitan/blob/3152b444b36843d4f4e8769643fb29b53f347e1f/hw/top_earlgrey/rtl/autogen/top_earlgrey.sv#L118-L119